js/05/main.js

```diff
@@ -0,0 +1,47 @@
+// First remove all the punctuations and change the string to array and count the number of words in the array
+let text = 'I love teaching and empowering people. I teach HTML, CSS, JS, React, Python.';
+const words = text.split(' ');
+console.log(words);
+console.log(words.length);
+
+//In countries array check if 'Ethiopia' exists in the array if it exists print 'ETHIOPIA'. If it does not exist add to the countries list and print the array.
+let ethiopia = 'Ethiopia';
+if (countries.includes(ethiopia)) {
+    let index = countries.indexOf(ethiopia);
+    countries[index] = 'ETHIOPIA';
+} else
+    countries.push(ethiopia);
+console.log(countries);
+
+//In the webTechs array check if Sass exists in the array and if it exists print 'Sass is a CSS preprocess'. If it does not exist add Sass to the array and print the array.
+let sass = 'Sass';
+webTechs.includes(sass) != false ? console.log('Sass is a CSS preprocess') : webTechs.push(sass); console.log(webTechs);
```

Comment:
webTechs.includes(sass) ?  will be enough, you can remove !=false ?
cond1 ? case of true : case of false